hydro_lang/src/viz/api.rs

```diff
@@ -87,13 +85,21 @@ impl<'a> GraphApi<'a> {
         config: HydroWriteConfig,
     ) -> Result<(), Box<dyn Error>> {
         match format {
-            GraphFormat::Mermaid => Ok(crate::graph::debug::open_mermaid(self.ir, Some(config))?),
-            GraphFormat::Dot => Ok(crate::graph::debug::open_dot(self.ir, Some(config))?),
-            GraphFormat::ReactFlow => Ok(crate::graph::debug::open_reactflow_browser(
-                self.ir,
-                None,
-                Some(config),
-            )?),
+            GraphFormat::Mermaid => Ok(crate::viz::debug::open_mermaid(self.ir, Some(config))?),
+            GraphFormat::Dot => Ok(crate::viz::debug::open_dot(self.ir, Some(config))?),
+            GraphFormat::Hydroscope => {
+                #[cfg(feature = "viz")]
```

Comment:
Won't `viz` always be enabled if we could call this API?

Reply:
true dat. will remove.